routes/api/users.js

```diff
@@ -143,5 +145,34 @@ router.patch("/avatars", upload.single("avatar"),auth,  async(req, res, next) =>
     return next(err);
   }
 
-} )
+})
+
+router.get("/verify/:verificationToken", async (req, res, next) => {
```

Comment:
ze wzgledu na to, ze route jest dosc jasna (`/verify/`) to mozesz smialo zrobic `token` zamiast `verificationToken`. Tak naprawde, moglbys zrobic route `/verify-email/...` <- to daloby Ci jasno 100% info o co chodzi (bo moze kiedys bedziesz weryfikowal dodatkowo cos innego? 🤔).
Wowczas...